README.md

```diff
@@ -206,11 +213,14 @@ that you have a 32bit version of Python installed.
 
 ### Set up your django environment
 
+Before running any of the commands below, you should have all of the following running: couchdb, redis, elasticsearch,
+and kafka. To install and run kafka, follow the instructions here: 
```

Comment:
It definitely didn't work without kafka running. I can take out the line but it would probably be better to figure out why that was the case if it's not supposed to be?